### PR TITLE
Add rinkeby network

### LINKED ISSUE #3: Add Rinkeby Network
Hardhat template support migration on Ropston network only. We need to add more options for the template to be more usable.
 You can check this link to see how to add new [networks](https://hardhat.org/tutorial/deploying-to-a-live-network.html). If you have another network in mind, feel free to add it. Please don't forget to check our [contribution guideline](https://github.com/ArabsInBlockchain/hardhat-template/blob/main/CONTRIBUTING.md).

### PR DESCRIPTION
**Pull request by ShabanGomaa**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,5 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 ROPSTEN_URL=https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
+RINKEBY_RPC_URL=https://eth-rinkeby.alchemyapi.io/v2/<YOUR-API-LEY>
+MNEMONIC=<YOUR MNEMONIC>
```
Comment (yehia67) [inline .env.example:6]: Add a new line please

File: .env.example [hunk 2]
```diff
@@ -1,3 +1,5 @@
 ETHERSCAN_API_KEY=ABC123ABC123ABC123ABC123ABC123ABC1
 ROPSTEN_URL=https://eth-ropsten.alchemyapi.io/v2/<YOUR ALCHEMY KEY>
 PRIVATE_KEY=0xabc123abc123abc123abc123abc123abc123abc123abc123abc123abc123abc1
+RINKEBY_RPC_URL=https://eth-rinkeby.alchemyapi.io/v2/<YOUR-API-LEY>
```
Comment (yehia67) [inline .env.example]: Please rename the variable to be same as `ROPSTEN_URL` >>> `RINKEBY_URL`

File: hardhat.config.ts [hunk 3]
```diff
@@ -29,6 +29,14 @@ const config: HardhatUserConfig = {
       url: process.env.ROPSTEN_URL || '',
       accounts: process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
     },
+    
+    rinkeby: {
+      url: process.env.RINKEBY_RPC_URL,
+      accounts: {
+        mnemonic: process.env.MNEMONIC,
+      },
```
Comment (yehia67) [inline hardhat.config.ts]: Please make the `url` and `accounts` property similar to Ropston object

**Review by yehia67 — CHANGES_REQUESTED** [review]

File: .env.example (hunk 1 above)
Comment (ShabanGomaa) [reply .env.example:6]: Done

File: .env.example (hunk 2 above)
Comment (ShabanGomaa) [reply .env.example]: Done

File: hardhat.config.ts (hunk 3 above)
Comment (ShabanGomaa) [reply hardhat.config.ts]: Done

**Comment by yehia67** [comment]
Great! Thank you @ShabanGomaa

**Pull request merged**